macaroonbakery/store.py

```diff
@@ -0,0 +1,36 @@
+# Copyright 2017 Canonical Ltd.
+# Licensed under the LGPLv3, see LICENCE file for details.
+
+import os
+
+
+class MemoryOpsStore:
+    def __init__(self):
+        self._store = {}
+
+    def put_opts(self, key, ops, time):
```

Comment:
s/put_opts/put_ops/

Also, please add a doc comment for MemoryOpsStore.

Reply:
done.